sound/soc/soc-pcm.c

```diff
@@ -2759,6 +2759,8 @@ int soc_dpcm_runtime_update(struct snd_soc_card *card)
 	mutex_unlock(&card->mutex);
 	return 0;
 }
+EXPORT_SYMBOL_GPL(soc_dpcm_runtime_update);
```

Comment:
The naming may need updated too to add snd_soc prefix. Check that it's also in the public header as well.

Reply:
@lgirdwood, this function is in soc-dpcm.h but there's a comment above that says "internal use only". Does that mean it was intended not to be used by modules?